scripts/ci/test_cut_ironclaw_release.py

```
@@ -283,5 +283,77 @@ def test_candidate_manifest_resolution_fails_closed_when_crate_missing(
+            release.ensure_stable_changelog_entry(root, self.STABLE)
+
+    def test_rc_labeled_entry_does_not_satisfy_the_stable_gate(self) -> None:
+        """`description="v1.2.0-rc.1"` contains `v1.2.0` as a substring; the
+        exact-attribute probe must still refuse the stable cut."""
+        with tempfile.TemporaryDirectory() as directory:
+            root = Path(directory)
+            self.write_changelog(
+                root,
+                '<Update label="2026-08-10" description="v1.2.0-rc.1">…</Update>\n',
+            )
+            with self.assertRaisesRegex(
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)